### PR TITLE
(Fix) Use web3 HttpProvider in dev environment

### PR DESCRIPTION
**Pull request by fvictorio**
Depends on #324.

If web3 is not injected and you are in the `development` environment (using `npm start`), try to connect directly to testrpc (`localhost:8545`).

I think this can be very useful during development, since the transactions are submitted automatically.

### CODE REVIEW

File: src/utils/blockchainHelpers.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ export function getWeb3(cb) {
 	if (typeof web3 === 'undefined') {
     // no web3, use fallback
     console.error("Please use a web3 browser");
+    if (process.env.NODE_ENV === 'development') {
```
Comment (15chrjef) [inline src/utils/blockchainHelpers.js]: extract to variable

Comment (fvictorio) [reply src/utils/blockchainHelpers.js]: Do you mean the condition in the `if`? Something like:

```js
const devEnvironment = process.env.NODE_ENV === 'development'
if (devEnvironment) {
```

Comment (15chrjef) [reply src/utils/blockchainHelpers.js]: yes exactly 👍

**Comment by fvictorio** [comment]
@15chrjef Change applied.

**Pull request merged**